bluepysnap/circuit_validation.py

```diff
@@ -12,6 +12,7 @@
 
 from bluepysnap import schemas
 from bluepysnap.config import Parser
+from bluepysnap.exceptions import BluepySnapValidationError as Error
```

Comment:
I'm on the fence if I want to use the alias `Error` here. 

I just did it to minimize the changes to the file, as `BluepySnapValidationError` is verbose enough to have black restructure the code quite a bit.

Any thoughts?

Reply:
Same thoughts, what about using `as ValidationError`? Would the changes be smaller?

Reply:
I'll give it a go.

Reply:
Fixed: created a `warning` shortcut function to reduce the impact and kept the longer form to follow the style of the other exceptions.

Reply:
Nice, I like it more.